api-reference/10 UI Components/dxDataGrid/1 Configuration/summary/calculateCustomSummary.md

```diff
@@ -123,6 +121,202 @@ A summary value calculation is conducted in three stages: *start* - the **totalV
 
 ---
 
+**calculateCustomSummary** is called before [calculateCellValue](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#calculateCellValue). If you want to create a new column and use its values in a custom summary, the following example demonstrates how to implement these functions:
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    const  calculateArea = (rowData) => {
```

Comment:
Why is this an arrow function, but calculateAreaSummary below is declared as a regular function?